module/solutions/helper.js

```diff
@@ -3481,6 +3551,11 @@ module.exports = class SolutionsHelper {
             description: 1,
             externalId: 1,
             name: 1,
+            entityType:1,
```

Comment:
@praveenKDass Since we are not using any pre-hook for formatting, please keep the formatting like below
name: 1,